ophyd/controls/signal.py

```diff
@@ -409,6 +413,29 @@ def report(self):
                 'pv': pv
                 }
 
+    @property
```

Comment:
We were going to make this not a property, for consistency with `read`? Feel free to disagree; I just didn't want that to be forgotten.